packages/filebrowser-extension/src/index.ts

```diff
@@ -86,6 +86,8 @@ namespace CommandIDs {
 
   export const openPath = 'filebrowser:open-path';
 
+  export const openUrl = 'filebrowser:open-url';
```

Comment:
Probably better to stick with URL as in the standard API

```suggestion
  export const openURL = 'filebrowser:open-url';
```

Reply:
Looks like we use more often `Url` than `URL` in the code base when combined in a variable name? For example with `baseUrl` and `kernelIconUrl`.

No strong opinion on that one, although it reads a little bit more easily I think. 

> standard API

What does the standard API refer to?

Reply:
Hey method like [URLSearchParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams). But indeed if there are other use cases already in the code happy to keep it like this.

Reply:
Good point. Maybe we could do a pass on the code base in a separate PR to unify all of this.